cocli/cmd/cotsCreate.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/veraison/corim/cots"
+)
+
+var (
+	cotsCreateCtsDirs    []string
+	cotsCreateCtsFiles   []string
+	cotsCreateOutputFile *string
+)
+
+var cotsCreateCmd = NewCotsCreateCmd()
+
+func NewCotsCreateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "create a CBOR-encoded concise-ta-stores instance from the supplied concise-ta-store objects",
+		Long: `create a CBOR-encoded concise-ta-stores instance from the supplied concise-ta-store objects,
+
+	Create a concise-ta-stores containing stores from the cts1.cbor and cts2.cbor files.
+
+	  cocli cots create --ctsfile=cts1.cbor --ctsfile=cts2.cbor
+	 
+	Create a concise-ta-stores containing stores from the cts directory.
+
+	  cocli cots create --cts=cts
+	`,
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := checkcotsCreateArgs(); err != nil {
+				return err
+			}
+
+			ctsFilesList := filesList(cotsCreateCtsFiles, cotsCreateCtsDirs, ".cbor")
+
+			if len(ctsFilesList) == 0 {
+				return errors.New("no CTS files found")
+			}
+
+			// checkCorimCreateArgs makes sure cotsCreateCotsFile is not nil
```

Comment:
Is this code comment relevant here?

Reply:
Thanks for spotting, its not relevant here, will remove it